lib/db/wopi.php

```diff
@@ -120,15 +120,6 @@ public function getPathForToken($fileId, $version, $token){
 			return false;
 		}
 
-		$owner = $row['owner_uid'];
-		$view = new \OC\Files\View('/' . $owner . '/files');
-		$path = $row['path'];
-
-		if (!$view->is_file($path)) {
-			throw new \Exception('Invalid file path.');
-		}
-
-		$editor = $row['editor_uid'];
-		return array('owner' => $owner, 'editor' => $editor, 'path' => $path);
+		return array('owner' => $row['owner_uid'], 'editor' => $row['editor_uid'], 'path' => $row['path']);
```

Comment:
I guess we can skip validation here. Can't remember why I put it in the first place.